src/StructuredLogger.Tests/Analyzers/MessageTaskAnalyzerTests.cs

```diff
@@ -0,0 +1,195 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Build.Logging.StructuredLogger;
+using Xunit;
+
+namespace Microsoft.Build.Logging.StructuredLogger.UnitTests
+{
+    /// <summary>
+    /// Minimal stub implementation of Task for unit testing.
+    /// </summary>
+    public class Task

```

Comment:
This is useless, as it doesn't extend/inherit tested code. It actually yields the tests incompilable